Tutorials/Heisenberg1d/heisenberg1d.py

```diff
@@ -50,11 +50,12 @@
 pars['Learning']={
```

Comment:
Is it a faster setting for Tutorials? Does it reach the same accuracy as before?

Reply:
Oh I just changed it from AdaMax to Sgd.

Reply:
It's better, we should just check that this is consistent with what is written in the webpage